packages/flutter/test/foundation/diagnostics_test.dart

```diff
@@ -1180,7 +1180,11 @@ void main() {
       ifNull: 'disabled',
     );
 
-    expect(present.toString(), equals('onClick: Closure: () => void'));
+    if (!identical(0, 0.0)) {
```

Comment:
can we just search for something that starts with `onClick:` and contains `Closure`? I don't think we care about the exact syntax, do we?